docs/parallel/openmp/reference/openmp-directives.md

```diff
@@ -629,10 +629,10 @@ The `single` directive supports the following clauses:
 - [copyprivate](openmp-clauses.md#copyprivate)
 - [nowait](openmp-clauses.md#nowait)
 
-The [master](#master) directive lets you specify that a section of code should be executed only on the master thread.
-
 For more information, see [2.4.3 single construct](../2-directives.md#243-single-construct).
 
+To specify that a section of code should only be executed on the master thread, use the [master](#master) directive instead.
```

Comment:
could you change to 'executed on the main thread'

Reply:
Should be updated now.